server/src/routers/FavoritesRouter/FavoritesRouter.js

```diff
@@ -13,25 +13,27 @@ router
             res.send(favorites).status(200);
         } catch (e) {
             console.error(e);
-            res.send(e).status(500);
+            res.send(e.message).status(500);
         }
     })
     .post("/", async (req, res) => {
         try {
-            await addFavorite(req.body.token, req.body.menuitemid, req.body.name);
+            const { token, menuitemid, name } = req.body;
+            await addFavorite(token, menuitemid, name);
             res.sendStatus(200);
         } catch (e) {
             console.error(e);
-            res.send(e).status(500);
+            res.send(e.message).status(500);
         }
     })
     .post("/delete", async (req, res) => {
         try {
-            await removeFavorite(req.body.token, req.body.menuitemid);
+            const { token, menuitemid } = req.body;
+            await removeFavorite(token, menuitemid);
             res.sendStatus(200);
         } catch (e) {
             console.error(e);
-            res.send(e).status(500);
+            res.send(e.message).status(500);
```

Comment:
Just a note, since you are sending a string on error now, I will need to modify the frontend so our `post` and `get` utilities don't try to access the now undefined `message` property of the string (instead of the error object).

Reply:
I can change this back, I just thought it'd be more informative for debugging and didn't realize it was functional.